bedrock/privacy/templates/privacy/privacy-day.html

```diff
@@ -1,142 +1,310 @@
       </div>
+      <nav id="tips-nav-direct">
+        <ol role="tablist" class="pager-tabs">
+          <li role="presentation">
+            <a role="tab" aria-controls="ask" aria-selected="true" href="#ask" id="tab-ask"><span>{{ _('1') }}</span> {{ _('ASK') }}</a>
+          </li>
+          <li role="presentation">
+            <a role="tab" aria-controls="learn" aria-selected="false" href="#learn" id="tab-learn"><span>{{ _('2') }}</span> {{ _('LEARN') }}</a>
+          </li>
+          <li role="presentation">
+            <a role="tab" aria-controls="act" aria-selected="false" href="#act" id="tab-act"><span>{{ _('3') }}</span> {{ _('ACT') }}</a>
+          </li>
+          <li role="presentation">
+            <a role="tab" aria-controls="discuss" aria-selected="false" href="#discuss" id="tab-discuss"><span>{{ _('4') }}</span> {{ _('DISCUSS') }}</a>
+          </li>
+        </ol>
+      </nav>
+    </div>{# /#share-nav-wrapper #}
+  </div>{# /.container #}
 
-      <div class="main">
-
-          <div class="block">
-            <h2>{{ _('Privacy Principles') }}</h2>
-            <p>
-            {% trans %}
-              We have a set of privacy principles that stem from the Mozilla Manifesto and
-              inform how we develop our websites, products and services.
-            {% endtrans %}
-            </p>
-            <p class="go"><a href="{{ url('privacy.principles') }}">{{ _('Learn about our privacy principles') }}</a></p>
-          </div>
-
-          <div class="block">
-            <h2>{{ _('Do Not Track') }}</h2>
-            <p>
-            {% trans %}
-              Do Not Track is a feature in Firefox that helps put you in control of the way your
-              information is collected and used online.
-            {% endtrans %}
-            {% trans %}
-              When enabled, Do Not Track will tell advertising networks and websites that you want
-              to opt-out of tracking for purposes like behavioral advertising.
-            {% endtrans %}
-            </p>
-            <p class="go"><a href="https://support.mozilla.org/kb/how-do-i-turn-do-not-track-feature">{{ _('Enable Do Not Track in your Firefox today') }}</a></p>
-          </div>
 
-          <div class="block">
-            <h2>Lightbeam</h2>
-            <p>
-            {% trans %}
-              A Firefox add-on, Lightbeam allows you to see the first and hidden third party sites
-              you interact with on the Web. Using interactive visualizations, it’s an innovative way
-              to see who’s tracking you online.
-            {% endtrans %}
-            </p>
-            <p class="go"><a href="{{ url('lightbeam.lightbeam') }}">{{ _('Download Lightbeam and shine a light on who’s watching you') }}</a></p>
+  <div id="page-content">
+    <div class="container">
+      <div id="tips-wrapper" class="pager-content">
+        <div class="tip pager-page default-page" id="ask" role="tabpanel" aria-labelledby="tab-ask">
```

Comment:
Likewise, I think things like `role="tabpanel" aria-labelledby="tab-ask"` are also generated by the plugin?